_includes/main.scss

```diff
@@ -150,8 +150,11 @@ img {
   color: $white;
   padding: 5px 20px 20px;
 
-  a:hover {
+  a {
```

Comment:
this was failing a11y contrast check, so opted to keep the link text same color as rest of text, but with red underline when hovered over, for minimal feedback